common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -1046,6 +1046,12 @@
     public static final QName MODIFY_OUTGOING_FILTER_RULES_RESPONSE = QName.get(E_MODIFY_OUTGOING_FILTER_RULES_RESPONSE, NAMESPACE);
     public static final QName CONTACT_BACKUP_REQUEST = QName.get(E_CONTACT_BACKUP_REQUEST, NAMESPACE);
     public static final QName CONTACT_BACKUP_RESPONSE = QName.get(E_CONTACT_BACKUP_RESPONSE, NAMESPACE);
+    
+    //HAB
+    public static final String E_HAB_ORG_UNIT_REQUEST = "HabOrgUnitRequest";
```

Comment:
can we rename request to HABOrgUnitRequest? just to be in sync.

Reply:
Fixed